corehq/apps/userreports/tasks.py

```diff
@@ -354,11 +354,12 @@ def _queue_chunk(indicators):
 def build_async_indicators(indicator_doc_ids):
     # written to be used with _queue_indicators, indicator_doc_ids must
     #   be a chunk of 100
-    for ids in chunked(indicator_doc_ids, 10):
-        _build_async_indicators(ids)
+    memoizers = {'configs': {}, 'adapters': {}}
+    for ids in chunked(indicator_doc_ids, ASYNC_INDICATOR_CHUNK_SIZE):
+        _build_async_indicators(ids, memoizers)
 
 
-def _build_async_indicators(indicator_doc_ids):
+def _build_async_indicators(indicator_doc_ids, memoizers):
```

Comment:
It might be nice to make this optional, or since it looks like we don't chunk anymore, these can be local to the function.

Reply:
Same as above(?)